src/models/stackedAreaChart.js

```diff
@@ -375,11 +375,15 @@ nv.models.stackedAreaChart = function() {
                 //If we are in 'expand' mode, force the format to be a percentage.
                 var valueFormatter = (stacked.style() == 'expand') ?
                     function(d,i) {return d3.format(".1%")(d);} :
-                    function(d,i) {return yAxis.tickFormat()(d); };
+                    tooltip.valueFormatter(); 
```

Comment:
The problem with this is that if using the interactive guideline, people should be doing chart.interactiveLayer.tooltip.valueFormatter() instead of chart.tooltip.valueFormatter(), so this is not really correct...   instead we should probably just have a variable to track the original value so we can switch back to it...